cypress/integration/Crosscutting/BackToTopTest/index.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="Cypress" />
+import { When, Then, And } from "cypress-cucumber-preprocessor/steps";
+
+When('user scrolls 600 px down', () => {
+    cy.scrollTo(0, 600);
```

Comment:
the number should not be hardcoded - please pass it as parameter

Reply:
done